tests/unit/synapseclient/core/unit_test_utils.py

```diff
@@ -100,6 +100,37 @@ def __init__(self, id_attr_name, id):
         assert utils.id_of(foo) == "123"
 
 
+# TODO: Add a test for is_synapse_id_str(...)
+# https://sagebionetworks.jira.com/browse/SYNPY-1425
+
+
+def test_get_synid_and_version():
```

Comment:
Nit: Not to over-engineer the solution, but in the future, these are technically different test cases.  For example, right now it would be harder to determine if Test 2 or Test 3 failed, but if you split them into separate tests OR use `pytest.parametrize` (it it makes sense to) then it'd be easier to navigate.